bakerydemo/templates/tags/gallery.html

```diff
@@ -1,12 +1,14 @@
 {% load wagtailimages_tags %}
 
 {% for img in images %}
-{% image img fill-285x200-c100 as img_obj %}
-<div class="col-sm-6">
-    <figure class="gallery-figure">
-        <img src="{{img_obj.url}}" class="img-responsive" />
-        <figcaption>{{ img.title }}</figcaption>
+{% image img fill-645x480-c100 as img_obj %}
```

Comment:
question: can this be deleted now you have an image tag on line 7?

Reply:
Good spot, thank you